.buildkite/scripts/steps/test/ftr_configs.sh

```diff
@@ -57,6 +57,16 @@ while read -r config; do
 
   start=$(date +%s)
 
+  if [[ "${USE_CHROME_BETA:-}" =~ ^(1|true)$ ]]; then
+    echo "USE_CHROME_BETA was set - using google-chrome-beta"
+    export TEST_BROWSER_BINARY_PATH="$(which google-chrome-beta)"
+
+    # download the beta version of chromedriver
+    export CHROMEDRIVER_VERSION=$(curl https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions.json -s | jq -r '.channels.Beta.version')
+    export DETECT_CHROMEDRIVER_VERSION=false
+    node node_modules/chromedriver/install.js --chromedriver-force-download
```

Comment:
Is there a good way to cache this in the VM image?

Reply:
We can play this operation as we build the image (grab the beta version, run the installer) however, I'm not sure if it would make for much saved time - this is quite a fringe case (and it takes 2 sec for the download).